workbench/runtime.py

```diff
@@ -387,6 +389,29 @@ def attr(self, attr_name):
 ID_MANAGER = ScenarioIdManager()
 
 
+class WorkBenchUserService(UserService):
+    """
+    An implementation of xblock.reference.user_service.UserService
+    """
+
+    def __init__(self, uid):
+        """
+        Initialize user
+        """
+        self._user = XBlockUser(
+            is_current_user=True,
+            emails=["user@example.com"],
+            full_name="XBlock User",
```

Comment:
It might be nice to have this reflect the user id (so that you can distinguish between different users in workbench).